languages/kotlin/exercises/concept/basics/.docs/introduction.md

```diff
@@ -0,0 +1,94 @@
+Kotlin is **statically typed** programming language developed by JetBrains. This means that type of variables is defined on compile-time.
+
+## Variables
+
+Similarly to other statically-typed programming languages type of each variable should be defined in compile time. Similarly to other **modern** programming languages you can avoid explicit type declarations where they can be interfered by compiler from context. Use this ability to make your code more readable. But remember that there are cases where it's better to specify type explicitly - e.g. in public APIs.
```

Comment:
```suggestion
Similarly to other statically-typed programming languages, the type of each variable should be defined at compile time. You can avoid explicit type declarations where they can be inferred by the compiler from their context.
```

I've suggested removing the following:

> Use this ability to make your code more readable. But remember that there are cases where it's better to specify type explicitly - e.g. in public APIs.

The reason for this is to keep the introduction as concise as possible. The introduction should only contain information that is needed to understand the fundamentals of the concept and solve the exercise. Extra information should be left for the `after.md`, which is perfect for the content I've suggested removing.

Reply:
Moved to `after.md`.